src/impl/peerconnection.cpp

```diff
@@ -586,6 +586,9 @@ void PeerConnection::dispatchMedia([[maybe_unused]] message_ptr message) {
 		if (auto track = it->second.lock())
 			track->incoming(message);
 	} else {
+		if (mTrackLines.size()==1) {
+			mTrackLines.front().lock()->incoming(message);
```

Comment:
`lock()` may return `nullptr` if the `Track` object has been destroyed. You need to check to check it otherwise it will crash as soon as the user releases the shared pointer:
```suggestion
			if (auto track = mTrackLines.front().lock())
			    track->incoming(message);
```